packages/ses/src/error/tame-v8-error-constructor.js

```diff
@@ -333,5 +364,7 @@ export const tameV8ErrorConstructor = (
     },
```

Comment:
The getter above on line 351 returns `OriginalError.prepareStackTrace`. This initially looked less safe to me, because you're not attenuating it above. But actually, the original `prepareStackTrace` remains encapsulated within this taming, and cannot escape, because the assignment on line 367 calls the setter, which replaces it with an attenuation. Yes?

Reply:
Correct. I'll add a comment to clarify.

Reply:
Cool. Resolving.